### ISSUE TITLE
Can I use NCBI to construct a reference genome

### ISSUE THREAD

**Issue by xiaoying-codeLab** [MGI-tech-bioinformatics/DNBelab_C_Series_HT_scRNA-analysis-software#130]
Can I use NCBI to construct a reference genome?

**Comment by lishuangshuang0616** [comment]
yes, the NCBI GTF sometimes displays the `gene_name` tag as `gene`. It is recommended to replace it with `gene_name`. 
Of course, the analysis can be done without adjustment, and the content of `gene_id` will be used for `gene_name`.

![image](https://github.com/user-attachments/assets/dde4c96a-cb64-49d9-9b52-ec3aa6175b8e)
